src/containerapp/azext_containerapp/_params.py

```diff
@@ -440,3 +440,36 @@ def load_arguments(self, _):
+
+    # Container App job
+    with self.argument_context('containerapp job') as c:
+        c.argument('name', name_type, metavar='NAME', id_part='name', help=f"The name of the Container Apps Job. A name must consist of lower case alphanumeric characters or '-', start with a letter, end with an alphanumeric character, cannot have '--', and must be less than {MAXIMUM_CONTAINER_APP_NAME_LENGTH} characters.")
+        c.argument('cron_expression', help='Cron expression. Only supported for trigger type "Schedule"')
+        c.argument('image', help="Container image, e.g. publisher/image-name:tag.")
+        c.argument('replica_completion_count', options_list=['--replica-completion-count', '--rcc'], help='Number of replicas that need to complete successfully for execution to succeed.')
+        c.argument('replica_retry_limit', help='Maximum number of retries before the replica fails.')
+        c.argument('replica_timeout', help='Maximum number of seconds a replica can execute.')
+        c.argument('parallelism', help='Maximum number of replicas to run per execution.')
+        c.argument('workload_profile_name', options_list=['--workload-profile-name', '-w'], help='The friendly name for the workload profile')
+        c.argument('min_executions', type=int, help="Minimum number of job executions that are created for a trigger, default 0.")
+        c.argument('max_executions', type=int, help="Maximum number of job executions that are created for a trigger, default 100.")
+        c.argument('polling_interval', type=int, help="Interval to check each event source in seconds. Defaults to 30s.")
+
+    with self.argument_context('containerapp job create') as c:
+        c.argument('system_assigned', help='System assigned identity.')
+        c.argument('trigger_type', help='Trigger type. Schedule | Event | Manual')
+        c.argument('user_assigned', help='User assigned identity.')
+
+    with self.argument_context('containerapp job', arg_group='Scale') as c:
+        c.argument('min_executions', type=int, help="Minimum number of job executions to run per polling interval.")
+        c.argument('max_executions', type=int, help="Maximum number of job executions to run per polling interval.")
+        c.argument('polling_interval', type=int, help="Interval to check each event source in seconds. Defaults to 30s.")
+        c.argument('scale_rule_type', options_list=['--scale-rule-type', '--srt'], help="The type of the scale rule.")
+
+    with self.argument_context('containerapp job stop') as c:
+        c.argument('job_execution_name', help='name of the specific job execution which needs to be stopped.')
+        c.argument('execution_name_list', help='comma separated list of job execution names.')
+
+    with self.argument_context('containerapp job execution') as c:
+        c.argument('name', id_part=None)
+        c.argument('job_execution_name', help='name of the specific job execution.')
```

Comment:
For every cli command, the `--name` should be used for the resource you operate. Here what you want to operate is container app job execution, then it would be better to make `name` for job execution and  something like `--job-name` for its parent resource